### ISSUE TITLE
nsis installer just sits there and does nothing

### ISSUE THREAD

**Issue by snene** [electron-userland/electron-builder#3942]
electron-builder version: 20.43.0

electron-updater version: 4.0.6

electron version: 4.2.3

I am creating nsis installer for windows using electron-builder. The build command 'build -p always --x64 --ia32' is successful and the installer exe is generated.

But when I run the installer, it just sits there and does absolutely nothing.

With version 19.46.9 of the electron-builder it used to work perfectly fine. Started seeing the issue only after the upgrade to 20.43.0

**Comment by snene** [comment]
Hi @develar , Any clue as to what could be going wrong? There is a similar issue #3939 reported for Linux

**Comment by byAbaddon** [comment]
Worked when I downgrade version to
electron version: 4.2.3
electron-builder version: 19.46.9

**Comment by snene** [comment]
Thanks for the update @byAbaddon !

**Comment by javierforero** [comment]
@byAbaddon @snene  experiencing similar issues on my end. how did you down grade? Was it just updating the `package.json` or downgrading the image? Please post steps. Thank you!

**Comment by snene** [comment]
@javierforero Yes, you can downgrade by updating the package.json.

**Comment by snene** [comment]
@byAbaddon @javierforero I don't know exactly how but I created a new terminal window and then ran the command to build the app again and it suddenly started working. So try that or may be just try restarting your system.

**Comment by byAbaddon** [comment]
@javierforero
One day i lost to play why this app not work.
I saw post on @snene and attempted to back old version on electron 
on 4.2.3  and electron-packager on 19.46.9.
1) I deleted node_modules folder.
2) set  package.joson    
   "devDependencies": {
      "electron": "^4.2.3",
      "electron-builder": "^19.46.9"
    }
"scripts": {  "package": "electron-builder" }
3)npm install
4)npm run  package
building sucsess!
This  work for me!
